backends/sqs/backend.go

```diff
@@ -231,7 +230,8 @@ func (s *Backend) CreateQueue(ctx context.Context, in *rpc.CreateQueueRequest) (
 	queueName := QueueIdToName(in.Id)
 	attributes := make(map[string]*string)
 	for k, v := range in.Attributes {
-		attributes[k] = &v
+		value := v
+		attributes[k] = &value
```

Comment:
this could also be `&in.Attributes[k]`. The issue here is that the for loop variables are shallow copies, so they can't be referenced across the loop.